packages/storybook/.storybook/main.js

```diff
@@ -1,6 +1,7 @@
 import remarkGfm from 'remark-gfm';
```

Comment:
Thanks for tackling this! My only suggestion would be to double check that TurboSnap is still working as expected. Looks like it was disabled for this build because of the changes to `main.js`. I'd be fine with doing that verification after completing this PR in whatever next PR is created that makes a component change.